pages/common/ngrep.md

```diff
@@ -0,0 +1,23 @@
+# ngrep
+
+> Grep traffic on a network.
+
+- Capture traffic of all interface:
+
+`ngrep -d any`
+
+- Capture traffic of a specific interface {{dev}}:
```

Comment:
I think it might be better to change `{{dev}}` to a concrete example like `eth0`. Users might be confused to other devices like '/dev/sda1' etc.